context.go

```diff
@@ -37,16 +37,13 @@ func NewContext(app *App, set *flag.FlagSet, parentCtx *Context) *Context {
 
 	if c.Context == nil {
 		ctx, cancel := context.WithCancel(context.Background())
+		signals := make(chan os.Signal, 1)
```

Comment:
Does this need to live outside of the closure?

Reply:
Fixed